backend/src/Designer/Infrastructure/GitRepository/AltinnAppGitRepository.cs

```diff
@@ -359,7 +359,8 @@ public string[] GetLayoutSetNames()
         public bool AppUsesLayoutSets()
         {
             string[] layoutSetNames = GetLayoutSetNames();
-            string layoutSetRelativePath = GetPathToLayoutSet(layoutSetNames[0]);
+            // Check for size of layoutsetnames to handle new apps that does not have a layoutsfolder under /App/ui/
+            string layoutSetRelativePath = GetPathToLayoutSet(layoutSetNames.Length > 0 ? layoutSetNames[0] : "some-non-existing-folder");
```

Comment:
Suggest something like this instead, to avoid using a random folder name:
```suggestion
           string layoutSetRelativePath = null;
            if (layoutSetNames.Length > 0) {
                layoutSetRelativePath = GetPathToLayoutSet(layoutSetNames[0]);
            }
            return (!string.IsNullOrEmpty(layoutSetRelativePath) && DirectoryExistsByRelativePath(layoutSetRelativePath)) &&
                !(layoutSetNames.Contains("layouts") && layoutSetNames.Length <= 1);
```